compiler/rustc_typeck/src/check/regionck.rs

```diff
@@ -269,7 +269,7 @@ impl<'a, 'tcx> RegionCtxt<'a, 'tcx> {
     ///
     /// Consider this silly example:
     ///
-    /// ```
+    /// ```ignore HELP (is that @ intentional?)
```

Comment:
`@` is *old* syntax for "garbage-collected pointer": https://pcwalton.github.io/2013/06/02/removing-garbage-collection-from-the-rust-language.html. Not sure how to keep the same semantics - maybe `Box<i32>` is close?